tests/testthat/test_ruleLengths.R

```diff
@@ -0,0 +1,58 @@
+test_data6 <- readRDS("testdata/test_rulelength_data/test_e2e_data_6.rds")
+test_data7 <- readRDS("testdata/test_rulelength_data/test_e2e_data_7.rds")
+test_data8 <- readRDS("testdata/test_rulelength_data/test_e2e_data_8.rds")
+test_data9 <- readRDS("testdata/test_rulelength_data/test_e2e_data_9.rds")
+
+test_that("check rule length for run of 6", {
+  
+  results6 <- plot_auto_SPC(test_data6, chartType = "XMR", plotChart = FALSE, runRuleLength = 6)
+  results7 <- plot_auto_SPC(test_data6, chartType = "XMR", plotChart = FALSE, runRuleLength = 7)
+  results8 <- plot_auto_SPC(test_data6, chartType = "XMR", plotChart = FALSE, runRuleLength = 8)
+  results9 <- plot_auto_SPC(test_data6, chartType = "XMR", plotChart = FALSE, runRuleLength = 9)
+  resultslist <- unlist(c(results6[22,17], results7[22,17], results8[22,17], results9[22,17]))
+  names(resultslist) <- NULL
+  
+  expect_equal(resultslist, c(FALSE, TRUE, TRUE, TRUE))
```

Comment:
Here, to make this _look_ more as one might expect, you could use `!c(TRUE, FALSE, FALSE, FALSE)`